apps/01PigLatin.js

```diff
@@ -7,12 +7,54 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+//'a'!=== 'A', so we want to return the string value to all lowercase, so we use string.prototype.toLowerCase so that our new word in Pig Latin will be consistent
+
```

Comment:
@idlemindss You're close. We use toLowerCase() to clean the input so that we only have to checkout lower case vowels. 

Also, please don't delete my comments next time. Just add to them.

And some new lines (pressing enter) in there would make them easier to read on here.

![finn](https://media.giphy.com/media/S0slBtHUTUfQY/giphy.gif)